user/common-tasks/disposablevm.md

```diff
@@ -153,6 +153,15 @@ $ qvm-open-in-vm @dispvm:online-dvm-template https://www.qubes-os.org
 
 This will create a new DisposableVM based on `online-dvm-template`, open the default web browser in that DisposableVM, and navigate to `https://www.qubes-os.org`.
 
+#### Example of RPC policies to allow this behavior
+
+In dom0, add the following line to `/etc/qubes-rpc/policy/qubes.OpenURL`
```

Comment:
It matters _where_ in the file you add this line, since the file is interpreted from top to bottom and stops at the first rule that applies.

Reply:
Specified to add this rule at the top of the file